DateTime.spec.ts

```diff
@@ -6,4 +6,24 @@ describe("DateTime", () => {
 		expect(model.DateTime.is(d))
 		expect(d).toBe("2020-12-31T23:59:59.000Z")
 	})
+	it("localize with arguments", () => {
+		const date = new Date("20 Jul 2019 10:30:40 GMT+2")
```

Comment:
It is nicer to store them in an array. That way adding more is easier. Do it like this:
```
const data = [
 [<value0>, <expected0>],
 [<value1>, <expected1>],
]
```
and then loop over the array and to the test in stead of echoing yourself.